### PR TITLE
tackled some class<?> newInstance depeated, and some code refactor

### PR DESCRIPTION
**Pull request by leechor**
Signed-off-by: licho <lecho.sun@gmail.com>

## What is the purpose of the change

tackle some deprecate and refactor some class member

## Brief changelog

1. tackled some class<?> newInstance repeated
2. refactor some code, some change some class member to `final static`

### CODE REVIEW

**Comment by AlbumenJ** [comment]
LGTM

@leechor thanks for your contribution

pls resolve conflicts with the latest master branch

File: dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java [hunk 1]
```diff
@@ -1,445 +1,445 @@
...
-        if (null == qualifiedName) {
-            return null;
-        }
-        
-        int i = qualifiedName.lastIndexOf('.');
-        return i < 0 ? qualifiedName : qualifiedName.substring(i + 1);
-    }
-
-}
+/*
```
Comment (AlbumenJ) [inline dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java:1]: Pls recover end-line format of this file

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java (hunk 1 above)
Comment (leechor) [reply dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java]: it's CRLF to LF problem. recover it already.

**Review by AlbumenJ — APPROVED** [review]
LGTM.

**Pull request merged**